pkg/apis/mxnet/v1beta1/types.go

```diff
@@ -45,9 +45,13 @@ type MXJob struct {
 type MXJobSpec struct {
 	// CleanPodPolicy defines the policy to kill pods after MXJob is
 	// succeeded.
-	// Default to Running.
+	// Default to All.
```

Comment:
Why are we changing the default here?

Reply:
The default is actually `All` according to `defaults.go`. I believe it was a mistake, so drop by to fix it.